packages/devtools_app/test/performance/timeline_events/legacy/legacy_event_processor_test.dart

```diff
@@ -135,7 +135,7 @@ void main() {
         'pid': 94955,
         'ts': 118039650802,
         'ph': 'B',
-        'args': {},
+        'args': <Object?, Object?>{},
```

Comment:
all these values for `args` should be `<String, Object?>`

Reply:
Thanks! Done